src/views/_proxied-dot-io/product-docs-landing/vault-content.js

```diff
@@ -0,0 +1,103 @@
+const vaultContent = {
+  pageTitle: 'Documentation',
+  pageSubtitle:
+    'Welcome to Vault documentation! Vault is an identity-based secret and encryption management system. This documentation covers the main concepts of Vault, what problems it can solve, and contains a quick start for using Vault.',
+  featuredCard: {
+    heading: 'Get Started',
+    image: {
+      src: require('./vault-get-started-diagram.png'),
```

Comment:
This is only reason this is a JS file and not a JSON file. Alternate here could be to upload this asset to Dato. (Incidentally, we need a high-res asset here, hence the `FPO` text!).

Reply:
Random curiosity, is it possible to to set the `img.src` property to the path of the image and then do an inline require where this property is used?